### DISCUSSION TITLE
Setting user initials in edit timestamps

### DISCUSSION THREAD

**Discussion by pamoroso** [Interlisp/medley#1837] [Q&A] [answered]
On my Medley installation on Linux I'd like to set the user initials to be used in edit timestamps.

Since my Linux username is `paolo`, the default value of `INITIALSLST` is `((PAOLO NIL NIL))`, the default of `INITIALS` is `NIL`, and no initials are inserted in timestamps. So I added to my initfile the `P` command `(SETQ INITIALS "PA")` but the values of `INITIALS` and `INITIALSLST` don't change and Medley still doesn't insert the initials. The other expressions in the initfile are evaluated.

What else should I do?

**Comment by pamoroso** [comment]
I added the command `(ADDVARS (INITIALSLST ("paolo" "Paolo" "PA")))` to my initfile and there's no change: `INITIALS` is still `NIL` and `INITIALSLST` still `((PAOLO NIL NIL))`. Here is my init file: [INIT.txt](https://github.com/user-attachments/files/17137609/INIT.txt)

**Reply by nbriggs** [reply]
Look at the code for SETINITIALS, it is probably doing an EQ check so doesn’t work on strings. The user name should be an atom, I think

**Comment by MattHeffron** [comment, marked as answer]
@pamoroso The change isn't in the `COMS` of the INIT.txt you linked above. (It isn't in that file at all.)
@nbriggs The `ADDVARS` he showed is what I use successfully. ("the names were changed to protect the innocent" :wink:) 
`SETINITIALS` uses `string-equal`

**Reply by pamoroso** [reply]
Doh, I forgot to save the initfile and uploaded a prior version. This is the updated version which works:  [INIT.txt](https://github.com/user-attachments/files/17145411/INIT.txt)

Now `INITIALS` is `"PA"` and `INITIALSLST` is `(("paolo" "Paolo" "PA") (PAOLO NIL NIL))`.

**Reply by MattHeffron** [reply]
I'm still puzzled about where the `(PAOLO NIL NIL)` is coming from. I don't get a similar entry on `INITIALSLST`.

**Reply by pamoroso** [reply]
I'm building Medley from source if that makes a difference.
